### PR TITLE
Adding Veterans United

### PR DESCRIPTION
**Pull request by Connor-Knabe**
## Add/Update/Remove <CompanyName>

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [ ] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -675,6 +675,7 @@ Additions to this document that are properly formatted will automatically be pus
 - [Vayu Technology](http://vayu.com.au) | Sydney, Australia; Kathmandu, Nepal | Short interview, general programming questions and short take home challenge.
 - [Venminder, Inc.](https://www.venminder.com) | Elizabethtown, KY; Louisville, KY | Initial phone screen to explain position. If candidate interested they get a take home assignment followed by a non-scripted in-person interview with team members to judge personality fit.
 - [Verve](https://verve.co/careers) | London, UK | An intentionally short, take home exercise that mirrors real project work and incorporates code review elements
+- [Veterans United Home Loans](https://www.veteransunited.com/careers/) | Columbia, MO | Phone screen, remote or in person pair-programming exercise, and multiple in person panel interview with developers and managers.
```
Comment (mccartney) [inline README.md]:
```suggestion
- [Veterans United Home Loans](https://www.veteransunited.com/careers/) | Columbia, MO | Phone screen, remote or in person pair-programming exercise, and multiple in person panel interview with developers and managers.
```

Comment (mccartney) [reply README.md]: Removing trailing space as per Travis' comments:
 >   678:222  warning  Remove trailing whitespace  no-trailing-spaces  remark-lint

Comment (mccartney) [reply README.md]: @Connor-Knabe

Comment (Connor-Knabe) [reply README.md]: I made that change and corrected grammar @mccartney thanks! 👍

**Pull request merged**